sidebars.ts

```diff
@@ -14,112 +14,57 @@ const sidebars: SidebarsConfig = {
         {
           type: "doc",
           id: "getting-started",
-          label: "Overview",
+          label: "Overview"
         },
         {
           type: "category",
-          label: "Guides & Concepts",
+          label: "Platform & Concepts",
```

Comment:
doesnt make sense to me that "beginner guides" are under "platform and concepts"

Reply:
They should be deleted 🙌🏻